definitions/npm/@sendgrid/mail_v6.x.x/flow_v0.30.x-/test_mail_v6.x.x.js

```diff
@@ -0,0 +1,6 @@
+import mail from '@sendgrid/mail';
+
+mail.setApiKey('myApiKey');
+mail.send({
+  from: {email: "test@example.com"},
+});
```

Comment:
You'll need to add some failing assertions before we can merge this in. Could you also add just a couple more tests for the required keys in `MailData` and `MailContent`, among the others above?